pkg/drivers/kic/oci/oci.go

```diff
@@ -554,6 +554,26 @@ func ListContainersByLabel(ctx context.Context, ociBin string, label string, war
 	return names, err
 }
 
+// ListImagesByContainerLabel returns all the images names with a specified container label
+func ListImagesByContainerLabel(ctx context.Context, ociBin string, label string) ([]string, error) {
+	rr, err := runCmd(exec.CommandContext(ctx, ociBin, "ps", "-a", "--filter", fmt.Sprintf("label=%s", label), "--format", "{{.Image}}"))
```

Comment:
this doesnt seem to catch any kic image if u delete minikube
```
14:07:31 medya/workspace/minikube
master_srikrishnabh ⌅1
$ minikube delete --all
🔥  Deleting "minikube" in docker ...
🔥  Removing /Users/medya/.minikube/machines/minikube ...
💀  Removed all traces of the "minikube" cluster.
🔥  Successfully deleted all profiles
14:07:40 medya/workspace/minikube
master_srikrishnabh ⌅1
$ docker ps -a
```

but in reality

```
$ docker images | grep kicbase
gcr.io/k8s-minikube/kicbase          v0.0.28                                                 c3db8807ea16   7 weeks ago     981MB
gcr.io/k8s-minikube/kicbase          v0.0.27                                                 f94e08eb0335   2 months ago    980MB
```

Reply:
how about instead of docker ps, we do docker images and then we filter for all kicbase repos

both gcr and docker hub

Reply:
Done.